zerver/openapi/zulip.yaml

```diff
@@ -11762,6 +11762,200 @@ paths:
+                    invitees will be added after accepting the invitation. If `stream_ids` is
+                    empty, invitee will not be added to any stream.
+
+                    **Changes**: Before Zulip 7.0 (feature level 180), specifying `stream_ids`
+                    as empty list resulted in an error.
+                  type: array
+                  items:
+                    type: integer
+                  example: [1, 2]
+              required:
+                - invitee_emails
+                - stream_ids
+            encoding:
+              stream_ids:
+                contentType: application/json
+
+      responses:
+        "200":
+          description: Success.
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: "#/components/schemas/JsonSuccessBase"
+                  - additionalProperties: false
+                    properties:
+                      result: {}
+                      msg: {}
+                      ignored_parameters_unsupported: {}
+                    example: {"msg": "", "result": "success"}
+        "400":
+          description: Bad request.
+          content:
+            application/json:
+              schema:
+                oneOf:
+                  - allOf:
+                      - $ref: "#/components/schemas/CodedError"
+                      - example:
+                          {
+                            "code": "BAD_REQUEST",
+                            "msg": "Insufficient permission",
+                            "result": "error",
+                          }
+                        description: |
+                          An example JSON error response for when the user doesn't have permission
+                          to send invitations.
+                  - allOf:
+                      - $ref: "#/components/schemas/CodedError"
+                      - example:
+                          {
+                            "code": "BAD_REQUEST",
+                            "msg": "You must specify at least one email address.",
+                            "result": "error",
+                          }
+                        description: |
+                          An example JSON error response for when no email address is specified.
+                  - allOf:
+                      - $ref: "#/components/schemas/CodedError"
+                      - example:
+                          {
+                            "code": "BAD_REQUEST",
+                            "msg": "Stream does not exist with id: 11. No invites were sent.",
+                            "result": "error",
+                          }
+                        description: |
+                          An example JSON error response for when any of the streams with the
+                          specified stream IDs does not exist.
```

Comment:
I just say, "... when any of the specified streams does not exist or the user does not have permission to access one of the targeted streams."